tasks/go.py

```diff
@@ -391,6 +411,40 @@ def is_excluded(pkg):
                         print("Skipping {} ({}) excluded in .wwhrd.yml".format(package, license))
                     else:
                         licenses.append("core,\"{}\",{}".format(package, license))
+
+    # Additional Licenses
+    for pkg in additional:
+        if pkg.startswith(GITHUB_DOMAIN):
+            url = "https://{}/master/".format(pkg.replace(GITHUB_DOMAIN, GITHUB_RAW_DOMAIN))
```

Comment:
What about repositories that use the `main` branch? If the goal is to make this work with arbitrary packages we should probably get the default branch